source/system/Numbers.ooc

```diff
@@ -6,10 +6,7 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-include stdlib, stdint, stddef, float, ctype, sys/types
-version(windows || apple) {
-	include limits
-}
```

Comment:
Was included for windows and apple here, and included for linux previously in `Exception.ooc`.